packages/flutter_tools/templates/app_shared/web/flutter.js.tmpl

```diff
@@ -0,0 +1,132 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file should be fully static, and not generated via a template at `flutter create` time, but injected automatically by flutter when doing `flutter build` or `flutter run`. That way, end users won't feel the temptation of touching it :)